grid-proxy/internal/explorer/db/postgres.go

```diff
@@ -377,7 +377,7 @@ func (d *PostgresDatabase) nodeTableQuery() *gorm.DB {
 	return d.gormDB.
 		Table("node").
 		Select(
-			"node.id",
+			"DISTINCT node.id",
```

Comment:
please add a comment to explain the need for distinct

Reply:
at the end needed to drop that and use a subquery with to node_gpu table and inner join it with the node table.

The problem with distinct was postgres requires a field that is distinctly selected to be the initial value in the order by statement and that works fine but corrupts the `available_for` filter since we have another order statement to have the nodes rented by this twin in the first of the response.